CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v11.0.0](https://github.com/voxpupuli/puppet-grafana/tree/v11.0.0) (2022-08-17)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-grafana/compare/v10.0.1...v11.0.0)
+
+**Breaking changes:**
+
+- Remove `address` param from `grafana_organization` [\#284](https://github.com/voxpupuli/puppet-grafana/pull/284) ([alexjfisher](https://github.com/alexjfisher))
+
+**Implemented enhancements:**
+
+- Add support for custom repository URL [\#291](https://github.com/voxpupuli/puppet-grafana/pull/291) ([SimonHoenscheid](https://github.com/SimonHoenscheid))
+- Add `organizations` property to `grafana_user` [\#286](https://github.com/voxpupuli/puppet-grafana/pull/286) ([alexjfisher](https://github.com/alexjfisher))
+- Fix performance of `grafana_user` [\#285](https://github.com/voxpupuli/puppet-grafana/pull/285) ([alexjfisher](https://github.com/alexjfisher))
+- Accept `Sensitive` `ldap_cfg` [\#282](https://github.com/voxpupuli/puppet-grafana/pull/282) ([alexjfisher](https://github.com/alexjfisher))
+- Accept `Sensitive` `cfg` [\#280](https://github.com/voxpupuli/puppet-grafana/pull/280) ([alexjfisher](https://github.com/alexjfisher))
+
+**Fixed bugs:**
+
+- Fix `grafana_user` and switch to using `flush` [\#283](https://github.com/voxpupuli/puppet-grafana/pull/283) ([alexjfisher](https://github.com/alexjfisher))
+- Set grafana.ini owner to root with packages [\#264](https://github.com/voxpupuli/puppet-grafana/pull/264) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Switch documentation to puppet strings [\#294](https://github.com/voxpupuli/puppet-grafana/pull/294) ([SimonHoenscheid](https://github.com/SimonHoenscheid))
+- Disable Arch Linux testing [\#292](https://github.com/voxpupuli/puppet-grafana/pull/292) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Since it's a major release anyway, should this be listed as a breaking change?

Reply:
updated it